internal/compiler/widgets/common/lineedit-base.slint

```diff
@@ -3,87 +3,86 @@
 
 export component LineEditBase inherits Rectangle {
     in-out property <string> placeholder-text;
-    in-out property <length> font-size <=> i-text-input.font-size;
-    in-out property <string> text <=> i-text-input.text;
+    in-out property <length> font-size <=> text-input.font-size;
+    in-out property <string> text <=> text-input.text;
     in-out property <brush> placeholder-color;
-    in-out property <bool> enabled <=> i-text-input.enabled;
-    in-out property <bool> has-focus: i-text-input.has-focus;
-    in-out property <InputType> input-type <=> i-text-input.input-type;
-    in-out property <TextHorizontalAlignment> horizontal-alignment <=> i-text-input.horizontal-alignment;
-    in-out property <bool> read-only <=> i-text-input.read-only;
-    in property <int> font-weight <=> i-text-input.font-weight;
+    in-out property <bool> enabled <=> text-input.enabled;
+    in-out property <bool> has-focus: text-input.has-focus;
+    in-out property <InputType> input-type <=> text-input.input-type;
+    in-out property <TextHorizontalAlignment> horizontal-alignment <=> text-input.horizontal-alignment;
+    in-out property <bool> read-only <=> text-input.read-only;
+    in property <int> font-weight <=> text-input.font-weight;
     in property <brush> text-color;
-    in property <color> selection-background-color <=> i-text-input.selection-background-color;
-    in property <color> selection-foreground-color <=> i-text-input.selection-foreground-color;
+    in property <color> selection-background-color <=> text-input.selection-background-color;
+    in property <color> selection-foreground-color <=> text-input.selection-foreground-color;
     in property <length> margin;
 
     callback accepted( /* text */ string);
     callback edited(/* text */ string);
 
     public function set-selection-offsets(start: int, end: int) {
-        i-text-input.set-selection-offsets(start, end);
+        text-input.set-selection-offsets(start, end);
     }
 
     public function select-all() {
-        i-text-input.select-all();
+        text-input.select-all();
     }
 
     public function clear-selection() {
-        i-text-input.clear-selection();
+        text-input.clear-selection();
     }
 
     public function cut() {
-        i-text-input.cut();
+        text-input.cut();
     }
 
     public function copy() {
-        i-text-input.copy();
+        text-input.copy();
     }
 
     public function paste() {
-        i-text-input.paste();
+        text-input.paste();
     }
 
-    min-height: i-text-input.preferred-height;
-    min-width: max(50px, i-placeholder.min-width);
```

Comment:
I'm guessing now, but... was this line removed because the real `LineEdit` components end up with a bigger minimum anyway?